recipe/conda_build_config.yaml

```diff
@@ -615,6 +615,9 @@ root_base:
 ruby:
   - 2.5
   - 2.6
+# Pin rust to <1.49 on osx-64 due to min deployment target req, cf. https://github.com/conda-forge/rust-feedstock/issues/73
+rust:     # [osx and x86_64]
+  - 1.48  # [osx and x86_64]
```

Comment:
In reality we'd only need `<1.49`. But I'm not sure how/if our tooling supports pinning expressions apart from `X.Y.*`.